src/static/js/movements.js

```diff
@@ -932,6 +977,15 @@ $(function() {
             }
         },
 
+        event_dates: async function(){
+            let result = await common.ajax_post("movement", "mode=eventlink&movementdate=" + $("#movementdate").val());
+            let dates = jQuery.parseJSON(result);
+            $.each(dates, function(i, v){
+                $("#eventlinkdates").append("<option value='" + v.ID + "'>" + format.date(v.STARTDATETIME) + " - " + format.date(v.ENDDATETIME) +
+                " " + v.EVENTNAME + " " + v.EVENTADDRESS + ", " + v.EVENTTOWN + ", " + v.EVENTCOUNTY + ", " + v.EVENTCOUNTRY + "</option>");
```

Comment:
Not seeing #eventlinkdates get cleared, so every time this runs it will append new dates?

Reply:
I did clear the #eventlinkdates twice in movements.js (I've changed the id to #event). first time in that method and second time when #eventlink is unchecked (it will put 0 in the database).
I think this one was an old commit.